core/src/main/scala/com/databricks/labs/remorph/intermediate/ddl.scala

```diff
@@ -59,6 +59,7 @@ case class StructField(name: String, dataType: DataType, nullable: Boolean = tru
 case class StructType(fields: Seq[StructField]) extends DataType
 case class MapType(keyType: DataType, valueType: DataType) extends DataType
 case object VariantType extends DataType
+case class ColumnDetail(name: String, dataType: DataType, nullable: Boolean = true, comment: Option[String] = None)
```

Comment:
nit: This should be moved to `extension.scala` Since all custom IR extensions are maintained here.

Reply:
done